metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -154,6 +155,26 @@ class Compilers(
         .asScala
     }.getOrElse(Future.successful(new CompletionList()))
 
+  def getLocation(
```

Comment:
Let's maybe keep to the current naming convention: `getLocation` -> `typeDefinition`